microprofile.jdt/com.redhat.microprofile.jdt.core/plugin.xml

```diff
@@ -21,6 +22,13 @@
             <command id="microprofile/java/hover"/>
        </delegateCommandHandler>
    </extension>
+  
+   <!-- Delegate command handler for Java project types -->
+   <extension point="org.eclipse.jdt.ls.core.delegateCommandHandler">
+      <delegateCommandHandler class="com.redhat.microprofile.jdt.internal.core.ls.ProjectTypesDelegateCommandHandler">
+            <command id="java/projectTypes"/>
```

Comment:
Rename to `microprofile/java/projectTypes` to follow the same pattern than other commends and to be sure that other commands (outside JDT Quarkus LS extensions) are unique (that's why commands starts with microprofile/)

Reply:
To prevent confusion, should I change the command to `microprofile/java/projectLabels`? 

In this PR, instead of "projectType", I'm thinking of replacing "type" with "label".

@fbricon @angelozerr what do you think about this idea?

Reply:
Yes please.